references/recognition/README.md

```diff
@@ -44,15 +44,16 @@ The JSON files must contain word-labels for each picture as a string.
 The order of entries in the json does not matter.
 
 ```shell
-labels = {
-    'img_1.jpg': 'I',
-    'img_2.jpg': 'am',
-    'img_3.jpg': 'a',
-    'img_4.jpg': 'Jedi',
-    'img_5.jpg': '!',
+{
```

Comment:
Perhaps we could add:
```shell
# labels.json
```
to make it clear?

Reply:
For the name of the file ? 
Yes if you want but it's already written inside the dataformat sooo as you wish 😄

Reply:
@Atomme1 
I think some like this:
To make it really clear :sweat_smile: 
```
# labels.json
{
    "img_1.jpg": "I",
    "img_2.jpg": "am",
    "img_3.jpg": "a",
    "img_4.jpg": "Jedi",
    "img_5.jpg": "!",
    ...
}
```

Reply:
Would you mind editing the PR then? :)